Sources/Views/ALKChatCell.swift

```diff
@@ -266,7 +314,14 @@ public final class ALKChatCell: SwipeTableViewCell, Localizable {
     public func update(viewModel: ALKChatViewModelProtocol, identity _: ALKIdentityProtocol?, placeholder: UIImage? = nil) {
         self.viewModel = viewModel
         let placeHolder = placeholderImage(placeholder, viewModel: viewModel)
-
+        
+        /// this is used to set the icon according to data recived in metadata
+        let isAgentApp = ALApplozicSettings.isAgentAppConfigurationEnabled()
+        
+        if isAgentApp == true,let device = viewModel.platformSource, let platformImage = platformIcons[device] {
+            platformImageView.image = platformImage
+        }
```

Comment:
you can convert the let isAgentApp & if check into if let

Reply:
removed isAgentApp and used ALApplozicSettings.isAgentAppConfigurationEnabled() directly inside the if block